07-routing-gossip.md

```diff
@@ -1136,6 +1157,200 @@ A->D's `update_add_htlc` message would be:
 And D->C's `update_add_htlc` would again be the same as B->C's direct payment
 above.
 
+## Liquidity Ads
+
+Liquidity ads allow nodes to announce their willingness to provide funding to
+other nodes for a fee. Sellers advertise various rates for the liquidity they
+are selling depending on the guarantees offered to buyers.
+
+1. subtype: `will_fund_lease_rates`
+2. data:
+    * [`will_fund_lease_rates_tlvs`:`lease_rates`]
+
+1. `tlv_stream`: `will_fund_lease_rates_tlvs`
```

Comment:
Note how a TLV stream is used here to allow sellers to offer various types of leases. Buyers will only read the lease types they support and ignore the lease types they don't support (as long as we odd numbers for the TLVs).